farm/metrics.py

```diff
@@ -8,10 +8,24 @@
 def simple_accuracy(preds, labels):
     # TODO: THIS HACKY TRY CATCH IS FOR GNAD
     try:
+        # nested lists with different lengths
```

Comment:
I'd prefer more simple ways of doing this. I think flattening both preds or labels as 1d and then doing accuracy should do the same without padding.

Reply:
Ok, good idea :+1: 
I will change it to 1d